content/blog/2023-01-24-aws/index.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: nlmixr2, tidyverse and RStudio on AWS
+author: 'Nicola Melillo and Hitesh Mistry'
+date: '2023-01-23'
+slug: []
+categories: [nlmixr2, AWS]
+tags: [AWS]
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Running large population PK/PD analyses on laptops and desktops often requires long computational times. This is quite tedious. In addition, when using parallel computing on your machine, it can slow it down for a while, creating further nuisances.  
+
+Outsourcing computation to the cloud is a solution to this problem. Among the various cloud providers, Amazon Web Service (AWS) is one of the most famous and used by industries in various fields. AWS elastic compute cloud (EC2) is a service that allows the user to easily create her/his own “machines”, called instances, with a certain hardware and software configuration. It is interesting to note that it is possible to scale up and down those instances whenever the user wants, by choosing the most suitable hardware configuration for a given analysis. Broadly speaking, the user can change the type of CPU, the number of cores (up to 192!) and ---
+title: nlmixr2 is here

```

Comment:
Are these yaml lines intentional?

Reply:
Hello Bill,

Thanks for the comments!

So, these yaml lines are not intentional, so, they need to be amended, thanks.